zebra-utils/Cargo.toml

```
@@ -1,8 +1,10 @@
 [package]
 name = "zebra-utils"
+version = "1.0.0-beta.25"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
+description = "Utilities for Zebra development, not for library or application users"
```

Nitpick: "Miscellaneous" and "utilities" don't actually mean much.

Library users could update the checkpoints, and end-users might use the hash lookup tool. So I'm not sure if we need to tell them not to use this crate.

```suggestion
description = "Developer tools for Zebra maintenance and testing"
```